client.go

```diff
@@ -555,14 +555,16 @@ func CaptureErrorAndWait(err error, tags map[string]string, interfaces ...Interf
 }
 
 // CapturePanic calls f and then recovers and reports a panic to the Sentry server if it occurs.
-func (client *Client) CapturePanic(f func(), tags map[string]string, interfaces ...Interface) {
+// If an error is captured, both the error and the reported Sentry error ID are returned.
+func (client *Client) CapturePanic(f func(), tags map[string]string, interfaces ...Interface) (err interface{}, errorID string) {
```

Comment:
Nevermind, I forgot this is what `recover()` returns: https://golang.org/pkg/builtin/#recover